aten/src/ATen/core/op_registration/op_registration.h

```diff
@@ -249,34 +249,6 @@ class CAFFE2_API RegisterOperators final {
      * >         .schema("my_op")
      * >         .catchAllKernel<decltype(my_kernel_cpu), &my_kernel_cpu>());
      */
-    template<class FuncType, FuncType* kernel_func>
-    // enable_if: only enable it if FuncType is actually a function
-    guts::enable_if_t<guts::is_function_type<FuncType>::value, Options&&> catchAllKernel() && {
```

Comment:
This is public API, don't change it. Also I don't think you get any binary size improvements from changing this one.